### DISCUSSION TITLE
Network Drive Mapping

### DISCUSSION THREAD

**Discussion by contactnat** [blakeblackshear/frigate#4225] [General]
Hi all, sorry for the noob question, I have searched and not been able to find what I am looking for.. 
I have Frigate running on its own Intel NUC, I have Truenas running in Proxmox on a seperate server which I have an NFS share setup with the following path on ip 192.168.1.209: /mnt/store/NVR/Frigate. 
how should I configure the  volumes?

    volumes:
      - /home/paul/frigate/config.yml:/config/config.yml:ro
      - /path/to/network/storage:/media/frigate
      - /home/paul/frigate:/db

**Comment by foureight84** [comment]
You could also mount NFS with docker drivers.

https://docs.docker.com/storage/volumes/#create-a-service-which-creates-an-nfs-volume
https://phoenixnap.com/kb/nfs-docker-volumes

**Reply by foureight84** [reply]
Another example https://blog.stefandroid.com/2021/03/03/mount-nfs-share-in-docker-compose.html
